docs/operations/metrics.md

```diff
@@ -86,6 +86,7 @@ Most metric values reset each emission period, as specified in `druid.monitoring
 |`subquery/fallback/unknownReason/count`|Number of subqueries which cannot be materialized as frames due other reasons.|This metric is only available if the `SubqueryCountStatsMonitor` module is included.| |
 |`query/rowLimit/exceeded/count`|Number of queries whose inlined subquery results exceeded the given row limit|This metric is only available if the `SubqueryCountStatsMonitor` module is included.| |
 |`query/byteLimit/exceeded/count`|Number of queries whose inlined subquery results exceeded the given byte limit|This metric is only available if the `SubqueryCountStatsMonitor` module is included.| |
+|`httpClient/channelAcquire/time`|Time in nannoseconds spent by the httpclient to acquire the channel.| |
```

Comment:
Sorry for being pedantic, but can you please make it more descriptive for the user and the operator? A few actionable items

* What does `httpclient` mean here
* What does channel mean here
* nannoseconds is misspelled 
* /httpClient/channelAcquire/time-> httpClient/channelAcquire/timeNs 
* What is the normal value of this metric?